src/styles/blocks/intro.scss

```diff
@@ -0,0 +1,105 @@
+.intro {
+  --w: 100%;
+  --position: 0;
+  --count: 3;
+
+  position: relative;
+  display: flex;
+  width: var(--w);
+  flex-basis: 390px;
+  overflow: hidden;
+  border-radius: 30px;
+
+
+  @include onNotebook() {
+    grid-area: 1 / 3 / 2 / 5;
+    border-radius: 30px 0 30px 0;
+  }
+
+  @include theDesign() {
+    grid-column: 9 / -1;
+  }
+
+  &__pictures {
+    width: calc(var(--w) * var(--count));
+    display: flex;
+
+    transform: translateX(calc((-1 * var(--position)) * var(--w)));
```

Comment:
🤯

Reply:
on the small viewport, I can see all links on the menu <img alt="Screenshot 2021-11-08 at 20 29 39" width="401" src="https://user-images.githubusercontent.com/47606379/140797716-01b99a45-71d2-4933-b945-3aa94757e5b0.png">

Menu available even on pagers :)